src/gui/processing/qgsprocessingmultipleselectiondialog.cpp

```diff
@@ -404,6 +405,17 @@ void QgsProcessingMultipleInputPanelWidget::populateFromProject( QgsProject *pro
       break;
     }
 
+    case QgsProcessing::TypePlugin:
```

Comment:
Would be good to extend TestProcessingGui::testMultipleFileSelectionDialog() to cover this too

Reply:
Done.